TrkFilters/src/HelixFilter_module.cc

```diff
@@ -59,7 +59,7 @@ namespace mu2e
         fhicl::Atom<double>             minNLoops            {     Name("minNLoops"),               Comment("minNLoops      ") };
         fhicl::Sequence<std::string>    helixFitFlag         {     Name("helixFitFlag"),            Comment("helixFitFlag   "), std::vector<std::string>{"HelixOK"} };
         fhicl::Atom<int>                debugLevel           {     Name("debugLevel"),              Comment("debugLevel")     , 0 };
-        fhicl::Atom<bool>               prescaleUsingD0Phi   {     Name("prescaleUsingD0Phi"),      Comment("prescaleUsingD0Phi") };
+        fhicl::Atom<bool>               prescaleUsingD0Phi   {     Name("prescaleUsingD0Phi"),      Comment("prescaleUsingD0Phi"), false };
```

Comment:
Is this no a deprecated option?  The general rule is that all config that could reasonably have different settings i a production job be set in prolog, not code default.

Reply:
The point of this PR is to have no defaults for the filters. This is not a deprecated parameter. It is used only in the `trkHelix_ipa_phiScaled` trigger path, that's why I thought it was better to have the default here instead of being forced to add 1 line in each HelixFilter module configuration used in each track trigger.

Reply:
I'll make it an `OptionalAtom` if that's ok

Reply:
done